### DISCUSSION TITLE
Add calendar events from calendar card

### DISCUSSION THREAD

**Discussion by Boringj** [totaldebug/atomic-calendar-revive#472] [Feature Requests]
Has anyone explored the idea of being able to add an event to google calendars from within the calendar card?  I want to replace our existing whiteboard calendar with my HA calendar.

**Comment by marksie1988** [comment]
This hasn't been looked at but would be interesting to see if it would be possible

**Comment by marksie1988** [comment]
duplicate: https://github.com/totaldebug/atomic-calendar-revive/discussions/1209
